sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/Internal/ValidationOptions.cs

```diff
@@ -60,13 +61,17 @@ public void Add(string connectionString)
             _hostKeyMappings.Add(host.Host, accessKey);
         }
 
-        internal bool ContainsHost()
+        public void Add(Uri endpoint)
```

Comment:
What do you expect ValidationOptions to do when adding only `Host` and `null` value? Might it be problem-leading when `TryGetKey` get a `null` `accessKey`?

Reply:
Works for AAD/TokenCredential where AbuseProtection still works, but no signature validation needs.